.vs/config/applicationhost.config

```diff
@@ -0,0 +1,1038 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No worries, I've also been busy with other projects so excuse my late reply. 
I don't think the file is needed (they seem to be VS environmental settings) so you can discard it.